docs/int/quickstart/update.md

```diff
@@ -14,15 +14,15 @@ To do this, follow these steps:
 ### Navigate to the node directory
 
 <Tabs groupId="group-solo">
-  <TabItem value="group" label="Group cluster" default>
+  <TabItem value="group" label="charon-distributed-validator-node" default>
     <pre>
       <code>
         cd charon-distributed-validator-node
       </code>
     </pre>
   </TabItem>
   
-  <TabItem value="solo" label="Solo cluster">
+  <TabItem value="solo" label="charon-distributed-validator-cluster">
```

Comment:
Should we add info on a clean shutdown before git pull?

Reply:
Do you need to shut down the containers before pulling an update? I would have thought, calling `docker compose up -d` again would have just detected what's different and made the necessary changes, no shutdown of e.g. the EL+CL if not required (which risks unclean shutdowns in itself)